### ISSUE TITLE
vocab size using validation set

### ISSUE THREAD

**Issue by ngoanpv** [iotayo/aivivn-tone#1]
Please let me know why we use validation set to build vocab instead of using training set? 

```python
def load_data_in_parts(train_src, train_tgt, val_src, val_tgt, batch_size=64, save_path=CHECKPOINT_PATH):
    # prepare dataset
    print("Reading data...")
    val = Seq2SeqDataset.from_file(val_src, val_tgt)
    print("Building vocab...")
    val.build_vocab(max_size=300)
    src_vocab = val.src_field.vocab
    tgt_vocab = val.tgt_field.vocab
......
```

**Comment by iotayo** [comment]
Hi ngoanpham,
The validation set and training set have the same vocab, so it's okay to use either one. I use the validation set because it has smaller size so build time is quicker.
